### PR TITLE
[17.12] changelog update for 17.12.1 rc1

### PR DESCRIPTION
**Pull request by jose-bigio**
Signed-off-by: jose-bigio <jose.bigio@docker.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,33 @@ be found.
...
+
+### Logging
+- awslogs: fix batch size calculation for large logs [moby/moby#35726](https://github.com/moby/moby/pull/35726)
+
+### Networking
+- Fix ingress network when upgrading from 17.09 to 17.12 [moby/moby#36003](https://github.com/moby/moby/pull/36003)
+* Add verbose info to partial overlay ID [moby/moby#35989](https://github.com/moby/moby/pull/35989)
+
+### Packaging
+- Set TasksMax in docker.service [docker/docker-ce-packaging](https://github.com/docker/docker-ce-packaging/pull/78)
```
Comment (andrewhsu) [inline CHANGELOG.md]: needs link to say `docker/docker-ce-packaging#78`

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,33 @@ be found.
...
+
+### Runtime
+* Bump Golang to 1.9.4
+* Bump containerd to 1.01
```
Comment (andrewhsu) [inline CHANGELOG.md]: `1.0.1`

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,33 @@ be found.
 **IMPORTANT**: You must stop all containers and plugins **BEFORE** upgrading to Docker CE 17.12.
 See related PR: [moby/moby#35812](https://github.com/moby/moby/pull/35812) 
 
+## 17.12.1-ce (2017-02-13)
```
Comment (andrewhsu) [inline CHANGELOG.md]: just have day as `DD`

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,33 @@ be found.
...
+- Fix ingress network when upgrading from 17.09 to 17.12 [moby/moby#36003](https://github.com/moby/moby/pull/36003)
+* Add verbose info to partial overlay ID [moby/moby#35989](https://github.com/moby/moby/pull/35989)
+
+### Packaging
+- Set TasksMax in docker.service [docker/docker-ce-packaging#78](https://github.com/docker/docker-ce-packaging/pull/78)
+
+### Runtime
+* Bump Golang to 1.9.4
+* Bump containerd to 1.0.1
+* Refresh containerd remotes on containerd restart [moby/moby#36173](https://github.com/moby/moby/pull/36173)
```
Comment (thaJeztah) [inline CHANGELOG.md]: nit: perhaps we should describe what it's fixing (e.g. "Fix dockerd not being able to reconnect to containerd when it is restarted")

**Pull request merged**